opm/common/OpmLog/LogBackend.cpp

```diff
@@ -27,18 +27,33 @@ namespace Opm {
     {
     }
 
+    LogBackend::~LogBackend()
+    {
+    }
 
-    bool LogBackend::includeMessage(int64_t messageFlag) {
-        if (((messageFlag & m_mask) == messageFlag) &&
-            (messageFlag > 0))
-            return true;
-        else
-            return false;
+    void LogBackend::configureDecoration(std::shared_ptr<MessageFormatterInterface> formatter)
+    {
+        formatter_ = formatter;
```

Comment:
> so if you want me to adopt a new point of view, point me to the links I should read :-)

Well - just like unnecessary braces it just looks ugly!  Seriously I have actually though a bit about your comments about the beauty of heterogeneity also in code - I kind of liked that view.